nbextensions/usability/collapsible_headings/main.css

```diff
@@ -78,10 +78,10 @@ div.cell {
   border-width: 0 1px 0 1px;
 }
 
-.chb div:hover,
-.chb .chb-hover {
-    border-color: #000;
+.chb div:hover, .chb .chb-hover, .jupyter-soft-selected .chb div{
+    border-color:  #42A5F5;
     border-left-color: transparent;
+	  border-width: 0 2px 0 0;
```

Comment:
This should be `border-width: 0 2px 0 2px;`, since the right (transparent border is used to give mitres to the brackets